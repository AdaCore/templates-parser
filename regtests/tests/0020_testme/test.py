from test_support import *

run('testme', ["testme38.tmplt"])
run('print_tree', ["testme38.tmplt"])
