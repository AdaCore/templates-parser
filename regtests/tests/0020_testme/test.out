Does it match? YES
 It definetely match !

[INFO] testme38.tmplt
[C_INFO]  1 FALSE
[TEXT] Does it match? @_YES_NO:MATCH(Test):TEST_VAR_@
[IF] @_MATCH(Test):TEST_VAR_@
  [TEXT]  It definetely match !
[ELSE]
  [TEXT]  Hmm... Contradiction!
[END_IF]
